### DISCUSSION TITLE
Styling TextFormBox

### DISCUSSION THREAD

**Discussion by AndriiHarashchak** [bdlukaa/fluent_ui#234] [Q&A] [answered]
Hello everybody. When using TextForm I have the ability to decorate input by setting the "Decoration" property. But TextFormBox doesn`t have such "feature". So I have a question: How can I set the border/border-radius of TextFormBox?

**Comment by bdlukaa** [comment, marked as answer]
You can set the `TextFormBox.decoration` property. Unfortunately the `.foregroundDecoration` property is not supported (see https://github.com/bdlukaa/fluent_ui/issues/574).
